specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-03-01-preview/componentLinkedStorageAccounts_API.json

```diff
@@ -0,0 +1,299 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ComponentLinkedStorageAccounts"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request when updating linked storage accounts for an Application Insights component. The updated linked storage account is returned.",
+            "schema": {
+              "$ref": "#/definitions/ComponentLinkedStorageAccounts"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ComponentLinkedStorageAccountsUpdate": {
+            "$ref": "./examples/LinkedStorageAccountsUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Delete linked storage accounts for an Application Insights component.",
+        "operationId": "ComponentLinkedStorageAccounts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StorageTypeParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request when deleting linked storage accounts for an Application Insights component."
+          },
+          "204": {
+            "description": "Successful request when deleting linked storage accounts for an Application Insights component."
+          }
+        },
+        "x-ms-examples": {
+          "ComponentLinkedStorageAccountsDelete": {
+            "$ref": "./examples/LinkedStorageAccountsDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describe the format of an Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
```

Comment:
Replace this with a reference to 'Resource' defined in common-types/resource-management/v1/types.json

Reply:
Fixed